### PR TITLE
Add org.gtk.Gtk3theme.Jade-1-Indigo

### PR DESCRIPTION
**Pull request by madmaxms**
new Gtk3theme: Jade-1-Indigo

### CODE REVIEW

**Comment by madmaxms** [comment]
bot build org.gtk.Gtk3theme.Jade-1-Indigo

File: org.gtk.Gtk3theme.Jade-1-Indigo.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+   "id":"org.gtk.Gtk3theme.Jade-1-Indigo",
+   "branch":"3.22",
+   "runtime":"org.freedesktop.Sdk",
+   "build-extension":true,
+   "sdk":"org.freedesktop.Sdk",
+   "runtime-version":"19.08",
```
Comment (bilelmoussaoui) [inline org.gtk.Gtk3theme.Jade-1-Indigo.json:7]:
```suggestion
   "runtime-version":"20.08",
```

Comment (madmaxms) [reply org.gtk.Gtk3theme.Jade-1-Indigo.json]: ok, done

**Comment by madmaxms** [comment]
bot, build org.gtk.Gtk3theme.Jade-1-Indigo

**Pull request closed without merging**